packages/gatsby-theme-api-docs/src/components/resource/method/responses.js

```diff
@@ -61,9 +61,14 @@ const Responses = ({ apiKey, responses, contentType }) => {
               <LinkContainer>
                 {responseDetails ? (
                   <SpacingsInline alignItems="center">
-                    {showDescription
-                      ? markdownFragmentToReact(response.description)
-                      : responseDetails}
+                    {showDescription ? (
+                      markdownFragmentToReact(response.description)
+                    ) : responseDetails ===
+                      response.body[contentType[index]].type ? (
+                      <p>{responseDetails}</p>
+                    ) : (
+                      responseDetails
```

Comment:
@gabriele-ct How would you write this piece of code? For me it feels like the nested conditions are still the best way to implement it. But I'm open for suggestions ;)

Reply:
In JSX it's really controversial. I still prefer to be explicit in the conditions to I'd split this in 3 different conditionals. However I had some pushback from Nikolaus about this approach, so I'm ok to keep nested ternary operators in JSX